src/calls.c

```diff
@@ -65,12 +64,27 @@
 #include <src/stats.h>                  /* statements */
 
 #include <src/saveload.h>               /* saving and loading */
-#include <src/hpc/tls.h>                /* thread-local storage */
 
 #include <src/vars.h>                   /* variables */
 
 #include <assert.h>
 
+
+void SET_NAME_FUNC(Obj func, Obj name)
+{
+    GAP_ASSERT(name == 0 || IS_STRING_REP(name));
+    FUNC_HEADER(func)->name = name;
+}
+
+Char * NAMI_FUNC(Obj func, Int i)
+{
+    return CSTR_STRING(ELM_LIST(NAMS_FUNC(func),i));
+}
```

Comment:
These two functions were moved from the header to the C file, so as to not force everybody who uses the header to also include `stringobj.h` and `plist.h`. Callers of this are not speed critical, so it's fine to not inline these.
Perhaps I should add a comment to explain this?

But perhaps it would be good to do this in a more systematic way, and e.g. change all those getters/setters to be non-inline? A matter of taste, I guess...